draft-ietf-ppm-dap.md

```diff
@@ -536,16 +536,18 @@ individual shares to each helper.
 
 ### HPKE Configuration Request {#hpke-config}
 
-Before the client can upload its report to the leader, it must know the public
-key of each of the aggregators. These are retrieved from each aggregator by
-sending an HTTP GET request to `[aggregator]/hpke_config?task_id=[task-id]`, where
-`[aggregator]` is the aggregator's endpoint URL, obtained from the task
-parameters, and `[task-id]` is the task ID obtained from the task parameters,
-encoded in Base 64 with URL and filename safe alphabet with no padding, as
-specified in sections 5 and 3.2 of {{!RFC4648}}. If the aggregator does not
-recognize the task ID, then it responds with HTTP error status 404 Not Found and
-an error of type `unrecognizedTask`. The aggregator responds to well-formed
-requests with status 200 and an `HpkeConfig` value:
+Before the client can upload its report to the leader, it must know the HPKE
+configuration of each aggregator. See {{compliance}} for information on HPKE configuration algorithms.
```

Comment:
Hmm, this doesn't seem to add anything new beyond "In the absence of an application profile standard specifying otherwise,..." What am I missing?

Reply:
My worry is that the current text says that the only option is X25519. Here we just point to {{compliance}}, which says:

> In the absence of an application profile standard specifying otherwise,
> a compliant DAP application MUST implement the following ciphersuite:

So if I'm interested in implementing DAP, my choice is to either use X25519 or get the WG to adopt an alternative "application profile standard".

Reply:
This might make sense in the end game, but at this early stage, I think this adds an unnecessary burden to implementors.

Reply:
Ah, I understand the disconnect now. An application profile is not something a WG has to adopt! On the contrary, it's just something a specific deployment of PPM can pick and then use as needed.

Reply:
Ohh I see. Perhaps adding some non-jargony language would help.

Reply:
Maybe we could say "application or deployment specific profile"?